utils/versionparser.go

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	VersionSeparator = "."
+	VersionLength    = 3
+)
+
+func ParseVersion(v string) ([]int, error) {
+	parts := make([]int, VersionLength)
+	split := strings.SplitN(v, VersionSeparator, VersionLength)
+	if len(split) != VersionLength {
+		return nil, fmt.Errorf("could not parse version '%s'", v)
+	}
+	// Remove 'v' from major version part
+	split[0] = strings.Replace(split[0], "v", "", 1)
```

Comment:
We can't just ignore the first char/rune as not all version numbers contain an `v` as a first character.